bin/node/runtime/src/lib.rs

```diff
@@ -757,6 +765,31 @@ impl pallet_bags_list::Config<VoterBagsListInstance> for Runtime {
 	type WeightInfo = pallet_bags_list::weights::SubstrateWeight<Runtime>;
 }
 
+/// A special VoterList instance used to test the StakeTracker.
+type VoterBagsListTrackerInstance = pallet_bags_list::Instance3;
```

Comment:
where's instance 1?

Reply:
https://github.com/paritytech/substrate/blob/bed5ad48b9be7e546878b68faef1620889cc2b2d/bin/node/runtime/src/lib.rs#L757-L756

Reply:
We need to maintain two separate VoterLists. One in Staking and another one in StakeTracker, hence the 3 instances. 
For that same reason VoterListTracker has a suffix, simply because the VoterList has already been taken.